### PR TITLE
fix: terrasos additional info

### PR DESCRIPTION
**Pull request by blushi**
## Description

Putting the terrasos metadata fields as part of the "known fields" so the corresponding label gets translated on the project additional info tab.
A longer term solution for translating "unknown fields" labels on the go will be addressed as part of a follow-up issue: https://regennetwork.atlassian.net/browse/APP-471

Also fixes "read more" translation

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by flagrede — APPROVED** [review]
Looks good!

File: web-marketplace/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx [hunk 1]
```diff
@@ -139,6 +147,16 @@ const ProjectPageMetadata: React.FC<React.PropsWithChildren<MetadataProps>> = ({
               metadata?.['@context'],
             )}
           />
+          <MetaDetail label={_(msg`region`)} value={bioregion} />
```
Comment (blushi) [inline web-marketplace/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx:150]: TODO confirm whether region is `regen:bioregion` or not before merging @S4mmyb @erikalogie

**Comment by blushi** [comment]
@erikalogie @S4mmyb you can test this on https://deploy-preview-2543--regen-marketplace.netlify.app/project/test-terrasos-project, switching to spanish

File: web-marketplace/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx (hunk 1 above)
Comment (blushi) [reply web-marketplace/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx]: it's different, using `regen:region` now

**Comment by erikalogie** [comment]
LGTM

**Comment by S4mmyb** [comment]
Looks great!

**Pull request merged**